src/content/release/breaking-changes/updated-material-3-progress-indicators.md

```diff
@@ -55,6 +55,21 @@ LinearProgressIndicator(
 ),
 ```
 
+To opt into the updated design spec for the `LinearProgressIndicator`, for the
+entire app, set the `ProgressIndicatorThemeData.year2023` flag to `false` in
+the `MaterialApp`:
+
+```dart highlightLines=2
+return MaterialApp(
+  theme: ThemeData(progressIndicatorTheme: const ProgressIndicatorThemeData(year2023: false)),
```

Comment:
I tried this out and it's saying the `year2023` parameter and corresponding property are deprecated?

Reply:
Yes, we added this flag with deprecation a few months ago.

Reply:
I see. It's a bit confusing though, as it's a deprecation warning, but in this case, you don't want to remove it.

Should we add an ignore comment to these snippets with an explanation of why the diagnostic is ignored?

Reply:
I didn't mean they should be ignoring the message but rather being aware of this migration steps. This flag is deprecated since we don't want people to think this will be a permanent flag. Adding message to ignore will defeat the purpose of deprecation.